README.md

```diff
@@ -9,9 +9,7 @@ Use `browzine-summon-adapter` to enhance Summon search results with BrowZine dat
 Visit Third Iron support to request your library API endpoint - http://support.thirdiron.com/
 
 ## What source code to use?
-Generally, use the source code found in `/dist/browzine-summon-adapter.js` because it has the broadest Internet Explorer support.
-
-However, if your institution has discontinued support for Internet Explorer and has instead focused on Microsoft Edge support, then feel free to use the source code directly found in `/src/browzine-summon-adapter.js` as it uses the latest JavaScript es6 features.
+Use `/src/browzine-summon-adapter.js`, we support IE11+ and evergreen browsers (Chrome, Firefox, Safari, Microsoft Edge).
```

Comment:
Closing the loop here @karlbecker. Looks like IE9 is the Windows ME of the Internet Explorer series. https://stackoverflow.com/a/5787944

Looks like code would work in IE8 and IE10, but not IE9 not too long ago. FWIW, it looks like some security issue specifically that a library could tweak if they really needed IE9 for some reason. But we may be good with IE10+ for now. 

Good point though and Thanks!

Reply:
Thanks for looking at their code - that's funny that they are supporting IE9 in kind of a non-traditional way!

And hey, since @johnseguin gave 👍 to only needing to support IE10 and above, works for me!  Happy to leave the old browsers in the dust. 😊 

We may want to mention what our goal of support is in the README, though.